src/web/components/Approvals/ApprovalItem.tsx

```diff
@@ -0,0 +1,40 @@
+import { z } from 'zod';
+
+import { ParticipantTypeSchema } from '../../../api/entities/ParticipantType';
+import { AvailableParticipantDTO } from '../../../api/participantsRouter';
+
+import './ApprovalItem.scss';
+
+type ParticipantItemProps = {
+  participant: AvailableParticipantDTO;
```

Comment:
As per previous comment, not sure if I should be using this DTO here or using a new one

Reply:
ParticipantRequestItem is now using ParticipantRequestDTO